src/component/DoctorList/DoctorListElement.tsx

```diff
@@ -0,0 +1,87 @@
+// map of the list of doctors for index
+import React from "react";
+import styled from "styled-components";
+import { TDoctorListProps } from "./index";
+import DoctorAvailabilities from "./DoctorAvailabilities";
+
+
+const Container = styled.ul`
+  diplay: flex;
+  margin-top: 5rem;
+`;
+
+const Row = styled.div`
+  &::after {
+    content: "";
+    clear: both;
+  }
+`;
+
+export interface Span {
+  span: string;
+}
+
+const Column = styled.div`
+  display: table;
+  float: left;
+  width: 100%;
+  @media only screen and (min-width: 1024px) {
+    ${(props: Span) => ``};
+    width: ${(props?: any) => (props.span ? (props.span / 12) * 100 : "8.33")}%;
+  } ;
+`;
+
+const Box = styled.li`
+  display: flex;
+  flex-wrap: wrap;
+  font-size: 1.1rem;
+  width: 60%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 1rem;
+  padding: 0.5rem;
+  border: 3px solid silver;
+  border-radius: 3px;
+  list-style: none;
+  @media only screen and (max-width: 768px) {
+    width: 100%;
+    font-size: 1rem;
+  } ;
+`;
+
+const Doctors: React.FC<TDoctorListProps> = ({ items }) => {
+  return (
+    <div>
+      <Container>
+        {items &&
+          items.map((item: any) => (
```

Comment:
doctors might be a better name
```suggestion
const Doctors: React.FC<TDoctorListProps> = ({ doctors }) => {
  return (
    <div>
      <Container>
        {doctors.map((item: Doctor) => (
```